lib/module.js

```diff
@@ -95,38 +96,83 @@ export default function SentryModule (moduleOptions) {
     }
   }
 
+  if (options.lazy) {
+    const defaultLazyOptions = {
+      injectMock: true,
+      injectLoadHook: false,
+      mockApiMethods: true,
+      chunkName: 'sentry',
+      webpackPrefetch: false,
+      webpackPreload: false
+    }
+
+    options.lazy = Object.assign({}, defaultLazyOptions, options.lazy)
```

Comment:
I'd prefer to have all options under the initial default object.
I guess you did it this way to shave some bytes when lazy loading is not used but I don't think it's worth it.
Also, it would make it problematic when annotating default options with types (which I have a local branch for).